cmd/zpool/zpool_main.c

```diff
@@ -5222,7 +5233,9 @@ zpool_do_iostat(int argc, char **argv)
 			 * every vdev, so skip this for histograms.
 			 */
 			if (((++cb.cb_iteration == 1 && !skip) ||
-			    (skip != verbose)) &&
+			    (skip != verbose) ||
+			    (!headers_once &&
+			    (cb.cb_iteration%winheight) == 0)) &&
```

Comment:
nit: please add a spaces around the `%` operator.  `cb.cb_iteration %  winheight`.

Reply:
Done